Rakefile

```diff
@@ -15,3 +15,9 @@ if defined?(RSpec)
     end
   end
 end
+
+desc "db:migrate + db:test:prepare"
+task :migrate => :environment do
+  Rake::Task["db:migrate"].execute
+  Rake::Task["db:test:prepare"].execute
+end
```

Comment:
Oops, I should've removed that. I added it to help me with development; it combines migrating the development database and updating the test DB so they're always in sync. I'll remove it; if it seems useful I can submit a separate pull request to add it.

On Sun, Nov 1, 2015 at 12:32 PM, Jesse Chan-Norris
<notifications@github.com> wrote:

>> @@ -15,3 +15,9 @@ if defined?(RSpec)
>>      end
>>    end
>>  end
>> +
>> +desc "db:migrate + db:test:prepare"
>> +task :migrate => :environment do
>> +  Rake::Task["db:migrate"].execute
>> +  Rake::Task["db:test:prepare"].execute
>> +end
> Can you explain this block of code?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/awesomefoundation/awesomebits/pull/202/files#r43585631